pkg/querier/queryrange/roundtrip.go

```diff
@@ -197,6 +199,7 @@ func NewMetricTripperware(
 	)
 
 	var c cache.Cache
+	tombstone := purger.NewTombstonesLoader(nil, nil)
```

Comment:
We don't have the `purger` config in `Loki` yet. Should we bring this config from cortex into this master update? 

Still, we can initialize purger with `nil` `DeleteStore`. Initializing purger with `nil` `DeleteStore` should be noop. 

@sandeepsukhani Hope I am correct here.